src/containers/record-modal.jsx

```diff
@@ -90,6 +91,9 @@ class RecordModal extends React.Component {
 
                 // update vmSound object with md5 property
                 vmSound.md5 = `${md5}.${vmSound.dataFormat}`;
+                // The VM will update the sound name to a fresh name
+                // if the following is already taken
+                vmSound.name = 'recording 1';
```

Comment:
It looks like we generally give names without spaces between word/number, and our automatic renamer doesn't handle whitespace (i.e. recording 1 and recording1 are distinct names)